src/title.js

```diff
@@ -0,0 +1,25 @@
+// LICENSE : MIT
+/*
+ * Gibook usage:
+ *
+ *      [import,title:<the title>](path/to/file)
+ */
+
+/* Get the specified <the title>
+ * @example:
+ *     getTitle(keyValObject)
+ * @param keyValObject
+ * @return {boolean,string}
```

Comment:
`{boolean|string}` is correct syntax.

- union type: [Use JSDoc: @param](http://usejsdoc.org/tags-param.html "Use JSDoc: @param")